components/network/tor/Makefile

```diff
@@ -72,3 +72,13 @@ REQUIRED_PACKAGES += SUNWcs
 REQUIRED_PACKAGES += system/library
 REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
 REQUIRED_PACKAGES += system/library/math
+# Auto-generated dependencies
```

Comment:
`REQUIRED_PACKAGES` above the generated section are duplicate to those in the generated section.

Reply:
Yes, that's right,
In my new version i fixed it.